### ISSUE TITLE
Using with A1111

### ISSUE THREAD

**Issue by sinand99** [layerdiffusion/sd-forge-layerdiffuse#111]
Since Forge is dead now, any plans to add support for A1111??

**Comment by Gushousekai195** [comment]
@lllyasviel I would be grateful if you ported this to A1111

**Comment by nitinmukesh** [comment]
Please support Auto1111

**Comment by Koxae** [comment]
I also join the request. Forge is very unstable.

**Comment by DiegoRRR** [comment]
Yes, please. (Recent versions of Forge don't work on my old system while last A1111 still does.)
A1111 doesn't use gradio4 yet though, can the fix of img2img work without gradio4?
